src/components/SelectProjectScreen/SelectProjectScreen.tsx

```diff
@@ -0,0 +1,93 @@
+import '../../index.css'
+import React, { useState } from 'react'
+import { Tooltip } from '@components/Tooltip/Tooltip'
+import NoAvatarProject from '@components/NoAvatar/NoAvatarProject'
+import Body2 from '@utils/typography/body2/body2'
+import Body1 from '@utils/typography/body1/body1'
+import H1 from '@utils/typography/h1/h1'
+import HelperText from '@utils/typography/helpertext/helpertext'
+import RadioButton from '@components/RadioButton/RadioButton'
+import { useGetProjects } from '@data-provider/query'
+import useScreenSize from '@hooks/useScreenSize'
+
+export interface SelectProjectScreenProps {
+  handleSelection: (id: string) => void
+}
+
+const SelectProjectScreen: React.FC<SelectProjectScreenProps> = ({
+  handleSelection
+}) => {
+  const { data, isLoading, error } = useGetProjects()
+  console.log(data, isLoading, error)
+  const [selectedProject, setSelectedProject] = useState<string | null>(null)
+  const screenSize = useScreenSize()
+
+  return (
+    <div className="flex flex-col max-w-[1032px] min-h-[500px] border border-primary-400 rounded-xl justify-center items-center bg-gray-600">
+      <div className="flex flex-col gap-4 items-center w-full p-6">
```

Comment:
I would move this line after the </H1> closing tag (line 39 I think) and I would close it after HelperText closing tag (line 86 probably). This change will make gap-10 style work and it will separate the title from the box.